dev/fattest.simplicity/src/componenttest/rules/repeater/MicroProfileActions.java

```diff
@@ -350,11 +348,13 @@ public class MicroProfileActions {
 
     private static final String[] STANDALONE10_FEATURES_ARRAY = { "mpContextPropagation-1.3",
                                                                   "mpGraphQL-2.0",
+                                                                  "mpMetrics-5.1",
                                                                   "mpReactiveMessaging-3.0",
                                                                   "mpReactiveStreams-3.0" };
 
     private static final String[] STANDALONE11_FEATURES_ARRAY = { "mpContextPropagation-1.3",
                                                                   "mpGraphQL-2.0",
+                                                                  "mpMetrics-5.1",
                                                                   "mpReactiveMessaging-3.0",
                                                                   "mpReactiveStreams-3.0" };
```

Comment:
It feels like moving `mpMetrics-5.1` out of the `MP70` feature lists and into the standalone lists is just making more work for ourselves.

A lot of the changes in `RepeatFaultTolerance` are only needed because we're moving `mpMetrics-5.1`.

I know that we've always had separate lists for standalone features (since #18453), but looking at it now, I can't see a good reason that the standalone features shouldn't be added to the regular feature sets - there doesn't appear to be a case where multiple versions of a standalone feature are compatible with the same microprofile version.